starter-code/iteration-1/main.js

```diff
@@ -0,0 +1,21 @@
+var myArray = [];
+function addTask(phrase) {
+    return myArray.push(phrase);
+}
+function listAllTasks() {
+    myArray.forEach(function (task) {
+        console.log(task);
+    });
+}
+function deleteTask(task) {
+    myArray.splice(myArray.indexOf(task), 1);
+    return myArray.length;
+}
+console.log("Number of items: 1", addTask('This is our first task'));
```

Comment:
aquí queda un poco redundante el "`number of items: 1`" por que la función devuelve ya el número de items.